pynq/lib/pynqmicroblaze.py

```diff
@@ -228,7 +224,7 @@ def run(self):
         self.state = 'RUNNING'
         self.reset.write(0)
 
-    def reset(self):
+    def reset_program(self):
```

Comment:
This change is due to the renaming of the `self.gpio` --> `self.reset`. Python does not allow the same name for attribute `reset` and the method `reset()`.